src/frontends/tensorflow/src/graph_iterator_proto.hpp

```diff
@@ -27,6 +29,11 @@ class GraphIteratorProto : public GraphIterator {
     std::vector<std::string> m_input_names;
     std::vector<std::string> m_output_names;
 
+    GraphIteratorProto()
+        : m_graph_def(std::make_shared<::tensorflow::GraphDef>()),
+          m_func_def(nullptr),
+          m_library_map() {}
+
```

Comment:
Please create a separate `GraphSavedModel` class in separate function

Reply:
Not sure I understand your point, please check in later update

Reply:
Done